modules/vaos/app/controllers/vaos/v2/appointments_controller.rb

```diff
@@ -113,22 +113,23 @@ def update
       # @raise [StandardError] For any unexpected errors during submission
       #
       def submit_referral_appointment
-        params = submit_params
+        patient_attrs = patient_attributes(submit_params)
         appointment = eps_appointment_service.submit_appointment(
           params[:id],
-          { referral_number: params[:referral_number],
-            network_id: params[:network_id],
-            provider_service_id: params[:provider_service_id],
-            slot_ids: [params[:slot_id]],
-            additional_patient_attributes: patient_attributes(params) }
+          { referral_number: submit_params[:referral_number],
+            network_id: submit_params[:network_id],
+            provider_service_id: submit_params[:provider_service_id],
+            slot_ids: [submit_params[:slot_id]] }.tap do |attrs|
+            attrs[:additional_patient_attributes] = patient_attrs if patient_attrs.present?
```

Comment:
Note: There is already code in the EPS service file to not include `additional_patient_attributes` when empty.